pkg/datagatherer/k8s/dynamic_test.go

```diff
@@ -129,50 +132,50 @@ func TestDynamicGatherer_Fetch(t *testing.T) {
 		"Foos in different namespaces should be returned if no namespace field is set": {
 			gvr: schema.GroupVersionResource{Group: "foobar", Version: "v1", Resource: "foos"},
 			objects: []runtime.Object{
-				getObject("foobar/v1", "Foo", "testfoo", "testns"),
-				getObject("foobar/v1", "Foo", "testfoo", "nottestns"),
+				getObject("foobar/v1", "Foo", "testfoo", "testns1"),
```

Comment:
For some reason, the new version of the client returns the objects ordered alphabetically by namespace.